Description: use HTTPS for paste.debian.net and paste.ubuntu.com
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2017-02-25

--- a/lib/App/Nopaste/Service/Debian.pm
+++ b/lib/App/Nopaste/Service/Debian.pm
@@ -1,7 +1,7 @@
 use strict;
 use warnings;
 package App::Nopaste::Service::Debian;
-# ABSTRACT: Service provider for Debian - http://paste.debian.net/
+# ABSTRACT: Service provider for Debian - https://paste.debian.net/
 
 our $VERSION = '1.007';
 
@@ -246,7 +246,7 @@
     "yaml" => "yaml",
 };
 
-sub uri { "http://paste.debian.net/" }
+sub uri { "https://paste.debian.net/" }
 
 sub fill_form {
     my $self = shift;
@@ -286,7 +286,7 @@
 
 =head1 NAME
 
-App::Nopaste::Service::Debian - Service provider for Debian - http://paste.debian.net/
+App::Nopaste::Service::Debian - Service provider for Debian - https://paste.debian.net/
 
 =head1 VERSION
 
--- a/lib/App/Nopaste/Service/Ubuntu.pm
+++ b/lib/App/Nopaste/Service/Ubuntu.pm
@@ -1,7 +1,7 @@
 use strict;
 use warnings;
 package App::Nopaste::Service::Ubuntu;
-# ABSTRACT: Service provider for Ubuntu - http://paste.ubuntu.com/
+# ABSTRACT: Service provider for Ubuntu - https://paste.ubuntu.com/
 
 our $VERSION = '1.007';
 
@@ -207,7 +207,7 @@
     "YAML"                           => "yaml",
 };
 
-sub uri { "http://paste.ubuntu.com/" }
+sub uri { "https://paste.ubuntu.com/" }
 
 sub fill_form {
     my $self = shift;
@@ -244,7 +244,7 @@
 
 =head1 NAME
 
-App::Nopaste::Service::Ubuntu - Service provider for Ubuntu - http://paste.ubuntu.com/
+App::Nopaste::Service::Ubuntu - Service provider for Ubuntu - https://paste.ubuntu.com/
 
 =head1 VERSION
 
--- a/t/004-02-debian.t
+++ b/t/004-02-debian.t
@@ -5,6 +5,6 @@
 
 use App::Nopaste::Service::Debian;
 
-is(App::Nopaste::Service::Debian->uri, 'http://paste.debian.net/');
+is(App::Nopaste::Service::Debian->uri, 'https://paste.debian.net/');
 
 done_testing;
--- a/t/004-10-ubuntu.t
+++ b/t/004-10-ubuntu.t
@@ -5,6 +5,6 @@
 
 use App::Nopaste::Service::Ubuntu;
 
-is(App::Nopaste::Service::Ubuntu->uri, 'http://paste.ubuntu.com/');
+is(App::Nopaste::Service::Ubuntu->uri, 'https://paste.ubuntu.com/');
 
 done_testing;
